lib/docker.js

```diff
@@ -212,7 +214,7 @@ function createDockerFile() {
     }
 
     if (!opts.deploy) {
-        contents += `RUN mkdir /opt/service\nADD . /opt/service\nWORKDIR /opt/service\nRUN ${npmCommand} install && npm dedupe\n`;
+        contents += `RUN mkdir /opt/service\nADD . /opt/service\nWORKDIR /opt/service\nRUN rm -rf node_modules && ${npmCommand} install && npm dedupe\n`;
```

Comment:
We don't do `rm -rf node_modules` here on purpose: we want developers to be able to quickly spin up their service

Reply:
Removed